### PR TITLE
Conformance test n7

### PR DESCRIPTION
**Pull request by manabuishii**
Check outputs type is "File" or not

### CODE REVIEW

File: core/handle.go [hunk 1]
```diff
@@ -75,6 +77,54 @@ func (h *Handler) Handle(job cwl.Parameters) error {
...
+			path = f.Name()
+			basename = filepath.Base(path)
+			location = fmt.Sprintf("file://%s", path)
+
+			defer f.Close()
+			fileinfo, _ := f.Stat()
+			size = fileinfo.Size()
+			h := sha1.New()
+			if _, err := io.Copy(h, f); err != nil {
+				log.Fatal(err)
```
Comment (otiai10) [inline core/handle.go:118]: should return error

File: core/handle.go [hunk 2]
```diff
@@ -75,6 +77,54 @@ func (h *Handler) Handle(job cwl.Parameters) error {
...
+			}
+
+			checksum = fmt.Sprintf("sha1$%x", string(h.Sum(nil)))
+		}
+		fmt.Println("{\"output_file\":{\"checksum\": \"" + checksum + "\",\"basename\": \"" + basename + "\",\"location\": \"" + location + "\",\"path\": \"" + path + "\",\"class\": \"File\",\"size\": " + strconv.FormatInt(size, 10) + "}}")
+	} else if h.Workflow.Outputs[0].Types[0].Type == "stdout" {
+		// TODO output file information, file name is random. so Any ?
+		// This is for n8
+		fmt.Println("{\"output_file\":{\"checksum\": \"sha1$47a013e660d408619d894b20806b1d5086aab03b\",\"basename\": \"output.txt\",\"location\": \"file:///tmp/out/output.txt\",\"path\": \"/tmp/out/output.txt\",\"class\": \"File\",\"size\": 13}}")
```
Comment (otiai10) [inline core/handle.go]: hard coded?

**Comment by otiai10** [comment]
please add `-n7` to this line and test
https://github.com/otiai10/yacle/blob/master/.travis.yml#L13

**Comment by otiai10** [comment]
~~It's another issue but does `-n4` pass the test? @manabuishii  Related https://github.com/otiai10/cwl.go/pull/28~~

Sorry, that's my misunderstanding. Please have a look at https://github.com/otiai10/yacle/commit/574d211447c11bfc054b2fb36cd9b357d85f189b

File: core/handle.go (hunk 2 above)
Comment (manabuishii) [reply core/handle.go]: sorry I need to remove this.

**Pull request merged**